components/feature/prompts/src/main/res/values/styles.xml

```diff
@@ -1,11 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <resources>
     <style name="MozTextInputLayout" parent="Widget.MaterialComponents.TextInputLayout.OutlinedBox">
         <item name="boxStrokeColor">@color/mozacBoxStrokeColor</item>
         <item name="boxStrokeWidth">2dp</item>
+        <item name="android:theme">@style/MozacPromptLoginTextInputLayoutCursorAppearance</item>
     </style>
     <style name="MozDialogStyle" parent="Theme.Design.Light.BottomSheetDialog">
         <item name="android:windowIsFloating">false</item>
         <item name="android:windowSoftInputMode">adjustResize</item>
     </style>
-</resources>
\ No newline at end of file
+    <style name="MozacPromptLoginTextInputLayoutCursorAppearance" parent="ThemeOverlay.MaterialComponents.TextInputEditText.OutlinedBox">
+        <item name="colorControlActivated">?mozacPromptLoginEditTextCursorColor</item>
```

Comment:
Where are we proving the value for `?mozacPromptLoginEditTextCursorColor`? As we don't have a default value what will happen for consumer that doesn't provide this value?

Reply:
Do you have a sample code where we are providing the value

Reply:
I use the attribute in Fenix in styles.xml like this:
```<item name="mozacPromptLoginEditTextCursorColor">@color/prompt_login_edit_text_cursor_color_normal_theme</item>```
The same for private mode.

Regarding the default value, I tested on Fenix and if I don't use at all 
`?mozacPromptLoginEditTextCursorColor` it works as before (the cursor color is the same as before - the color from the [Fenix issue](https://github.com/mozilla-mobile/fenix/issues/10113)). But I don't know what will happen for other apps.

Any suggestions? I'm open to any change.

Reply:
No problem, I just wanted to confirm that we don't have a crash when we don't provide a value. 👍